poc/vidpf.py

```diff
@@ -142,7 +142,10 @@ def eval(cls, agg_id, correction_words, init_seed, level, prefixes, cs_proofs, p
         # Compute the path verifier.
```

Comment:
Please expand on this comment here to explain what portion of the prefix tree we have path verifiability.